android/src/main/kotlin/com/jhomlala/better_player/BetterPlayer.kt

```diff
@@ -840,6 +844,8 @@ internal class BetterPlayer(
         exoPlayer?.release()
     }
 
+    val isLive get() = exoPlayer?.isCurrentMediaItemLive
```

Comment:
From what I understand, `plan limited` is just a banner cover on the player while in PIP mode on Android. I think `isCurrentMediaItemLive` should still be true because the datasource doesn't change like on iOS. However, keeping the original name is also OK, thanks.

Reply:
@mvn-datnguyen-hn 
Oh sorry.
I had forgotten the code is for Android. You are right :bow:

And thank you for updating the name. It is ok. 👍